lambdas/common/src/util/stringify-error.ts

```diff
@@ -1,3 +1,8 @@
 export function safeStringifyError(error: unknown) {
-  return error instanceof Error ? error.name : typeof error;
+  const errorType = error instanceof Error ? error.name : typeof error;
+
+  if (process.env.LOG_FULL_ERRORS === "true") {
```

Comment:
Can we you log/throw in the areas that this won't capture. As we saw on friday, `const errorMessage = `Following SSM parameters do not exist: ${errors.join(", ")}`;` this will get suppressed even with this new logic you've added. 

Parameter names are not sensitive. 

The types of logging that would be considered sensitive is like PII, API keys, etc.

Reply:
I agree with both of you, in the case of parameter retrieval i.e

```
  if (errors?.length) {
    const errorMessage = `Following SSM parameters do not exist: ${errors.join(", ")}`;
    throw new Error(errorMessage);
  }
```
I suppose we could of a log message here this should be safe, even with the flag LOG_FULL_ERRORS  not enabled from issue @barnabycollins mentioned other places where we are absolutely sure, we can follow the same pattern